app/routes/wow.ilvl-shopping-list.tsx

```
@@ -0,0 +1,368 @@
+    return json({
+      ...(await result.json()),
+      sortby: 'price',
+      formValues: { ...validatedFormData.data, desiredStats }
+    })
+  }
+
+  return json({})
+}
+
+type LoaderResponseType =
+  | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)